app/scripts/modules/netflix/fastProperties/domain/manualJudgementStage.ts

```diff
@@ -0,0 +1,26 @@
+import {IStage} from '../../../core/domain/IStage';
+import PropertyCommand from './propertyCommand.model';
+import {IUser} from '../../../core/authentication/authentication.service';
+
+export default class ManualJudgementStage implements IStage {
+  name: string;
+  type: string;
+  refId: string;
+  requisiteStageRefIds: string[] = [];
+  notifications: any[] = [];
+  judgmentInputs: any[] = [];
+  failPipeline: boolean = true;
+  instructions: string = 'Is Fast Property good to move forward';
```

Comment:
Maybe add a question mark at the end here, i.e. 'Is Fast Property good to move forward?'